Maths/Fibonacci.js

```diff
@@ -134,23 +165,32 @@ const FibonacciMatrixExpo = (n) => {
   // or                  F(n, n-1) = A * A * F(n-2, n-3)
   // or                  F(n, n-1) = pow(A, n-1) * F(1, 0)
 
-  if (n === 0) return 0
+  if (n === 0 || n === 0n) return n
+
+  const sgn = n < 0
+  if (sgn) n = -n
```

Comment:
For consistency, please use `n *= -1` here or use `number = -number` above as well.

Reply:
I would have to use `n *= -ONE` to avoid an error if `n` is `BigInt`. And I agree that the param naming is inconsistent